content/uiux/concepts/bias-in-user-research/bias-in-user-research.md

```diff
@@ -0,0 +1,111 @@
+
+To obtain accurate, useful data, researchers must be aware of their own biases. Researchers can also work to prevent participants from displaying common biases by strategically designing their studies. When analyzing results, researchers should consider how bias may have impacted the data, especially if the data is self-reported. 
+
+### Acquiescence Bias 
+
+ __Acquiescence bias__ is the tendency to agree with the interviewer. In general, participants are more likely to agree than to disagree. Thus, the question should not imply that there is a "correct" answer. Instead of asking for agreement or disagreement to a predefined statement, researchers can ask users to explain their point of view.  
+
+<details open> 
+
+<summary>Example</summary> 
+
+<br> 
+
+*“Would you agree that you are satisfied with the current interface?”* This question implies that the user is satisfied with the interface and could lead to acquiescence bias. <br>    
+
+*“How do you feel about the current interface?”* This phrasing is open-ended, so it does not lead participants to agree with a particular sentiment. </details> 
+
+### Confirmation Bias 
+
+ __Confirmation bias__ is the tendency for researchers to interpret data in a way that supports their opinions and beliefs. Researchers know the study’s goal, so they might make connections that aren’t explicit in the data set based on their opinions, beliefs, or expectations on how that goal should be met.  
+
+<details open> 
+
+<summary>Example</summary> 
+
+<br> 
+
+This survey asked, “Which of the following emotions do you feel when at the airport? Please select the options below.”  
+
+![Bar Chart that displays the data from the survey question: Which of the following emotions do you feel at the airport?](media/confirmation-bias-chart.png) 
+
+The team’s larger goal is to make a product that boosts positive feelings at the airport. Confirmation bias might mean looking at this data and only reporting findings based on the statistic “57.1 of our participants are excited when traveling at the airport.” While this is true, it ignores the other data and emotions, that 42.9% also feel anxious, confused, and frustrated.  </details> 
+
+### Demand Characteristics Bias 
+
+__Demand characteristics bias__ is a change in how a user answers questions due to the knowledge that they are part of a research study. If the user knows, or thinks they know, the objective of a study, this could influence their responses and behaviors. 
+
+<details open> 
+
+<summary>Example</summary> 
+
+<br> 
+
+A researcher runs a concept test to evaluate an early design idea. The participant may believe that the goal is to validate the idea so the company can move on to the development phase. This belief could influence them to try to “help” the researcher. They might provide positive feedback even if they don’t like the idea. </details> 
```

Comment:
```suggestion
```html
<details open> 

<summary>Example</summary> 

<br> 

A researcher runs a concept test to evaluate an early design idea. The participant may believe that the goal is to validate the idea so the company can move on to the development phase. This belief could influence them to try to “help” the researcher. They might provide positive feedback even if they don’t like the idea. </details> 
```
```